src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -6258,7 +6262,15 @@ public object VisitInvokeMemberExpression(InvokeMemberExpressionAst invokeMember
             var memberNameAst = invokeMemberExpressionAst.Member as StringConstantExpressionAst;
             if (memberNameAst != null)
             {
-                return InvokeMember(memberNameAst.Value, constraints, target, args, invokeMemberExpressionAst.Static, false);
+                return InvokeMember(
+                    memberNameAst.Value,
+                    constraints,
+                    target,
+                    args,
+                    invokeMemberExpressionAst.Static,
+                    propertySet: false,
+                    invokeMemberExpressionAst.NullConditional
+                    );
```

Comment:
Please move the closing parentheses to the end of previous line.

Reply:
Fixed